OpenEdXMobile/src/main/java/org/edx/mobile/whatsnew/WhatsNewItemModel.java

```diff
@@ -0,0 +1,82 @@
+package org.edx.mobile.whatsnew;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+
+import java.util.List;
+
+public class WhatsNewItemModel implements Parcelable {
+    public static final String PLATFORM_ANDROID = "android";
+    public static final String PLATFORM_IOS = "ios";
```

Comment:
`PLATFORM_ANDROID` & `PLATFORM_IOS` can be moved to an enum like `SupportedPlatforms`.